dojo/settings/settings.dist.py

```diff
@@ -451,7 +451,8 @@
         }
     }),
     # When enabled, force the password field to be required for creating/updating users
-    DD_REQUIRE_PASSWORD_ON_USER=(bool, True)
+    DD_REQUIRE_PASSWORD_ON_USER=(bool, True),
+    AZURE_DEVOPS_CACHE_DIR=(str, "/run/defectdojo"),
```

Comment:
Where and what is this variable used for?

Reply:
is used by the azure devops sdk to manage the cache.  [azure-devops-python-api](https://github.com/microsoft/azure-devops-python-api). The idea is to leave it as default because it fails if it is not there.